examples/2d/bar_waves/pylithapp_petsc.cfg

```diff
@@ -0,0 +1,132 @@
+[pylithapp.metadata]
+# This is not a self-contained simulation configuration file. This
+# file only specifies the general parameters common to the simulations
+# in this directory.
+keywords = [example, 2D, bar]
+features = [
+    Dynamic simulation,
+    Implicit-explicit formulation,
+    Quadrilateral cells,
+    Runge-Kutta time stepping,
+    pylith.meshio.MeshIOPETSc,
+    pylith.problems.TimeDependent,
+    pylith.materials.Elasticity,
+    pylith.materials.IsotropicLinearElasticity,
+    spatialdata.spatialdb.UniformDB,
+    pylith.meshio.DataWriterHDF5,
+    spatialdata.units.NondimElasticDynamic
+    ]
+
+# ----------------------------------------------------------------------
+# journal
+# ----------------------------------------------------------------------
+# Turn on some journals to show progress.
+[pylithapp.journal]
+device = color-console
+
+[pylithapp.journal.info]
+timedependent = 1
+solution = 1
+meshiopetsc = 1
+
+[pylithapp.journal.debug]
+timedependent = 0
+
+# ----------------------------------------------------------------------
+# mesh_generator
+# ----------------------------------------------------------------------
+[pylithapp.mesh_generator]
+# Set the reader to match the type of mesh file.
+reader = pylith.meshio.MeshIOPETSc
+#reader.filename = quad_200m.exo
```

Comment:
```suggestion
```

Reply:
I added back (and commented out) the Cubit reader lines. I was thinking the user could switch between the Cubit or PETSc reader. Would it be better to make a separate pylithapp.cfg file for each reader?

Reply:
We usually put settings that can apply to all simulations in a directory but different meshes in files like `petscmesh_quad.cfg`. These would override the settings in `pylithapp.cfg`. If they apply to a specific simulation, then we put them in something like `pwave_quad.cfg`.